Project-1/Peihong_Xie_Project_1_50375781.md

```diff
@@ -0,0 +1,299 @@
+  
+  O ≡ ∃parent_only¯.Person
+
+(e)  T that says that x has exactly two brothers. 
+
+  **p3m** (be a parent of exactly three male children) ≡ (≥3 parent_of. Male) ⊓ (≤3 parent_of. Male)
+
+  **p1** (be a parent of at least one child) ≡ ≥1 parent_of. Person -- the inverse of p1 is just "be a child of"!
+
+  T ≡ ∃p3m¯.Person ⊔ (¬M ⊓ ∃p1¯. (≥2 parent_of. Male ⊓ ≤2 parent_of. Male)) 
+
+  -- This means, either x is one of exactly 3 boys produced by a person (so x has exactly two brothers), or x is female and is a child of a person who has exactly two boys (so x has also exactly two brothers)
+
+[5] Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+
+Answer (based on the approach 2 in Question 4):
+
+$K_1 = (T_1,A_1)$ such that:
+
+T-box $T_1$={B (brother) ≡ M ⊓ ∃p2¯.Person, 
+
+O ≡ ∃parent_only¯.Person, 
+
+$GP (grandparent) ≡ ∃parentOf.(∃parentOf.M\sqcup ¬M)$}
+
+A-box $A_1$={(Mary,Karl):ParentOf, Karl:Male}.
+
+The following is a model $\Im=(\bigtriangleup^\Im,.^\Im)$ of $K_1$:
+
+$\bigtriangleup^\Im$={a,b,c}
+
+$M^\Im$={a}
+
+$B^\Im=\emptyset$
+
+$O^\Im$={b,a}
+
+$GP^\Im$={c}
+
+$parentOf^\Im$={(c,b),(b,a)}
+
+$Karl^\Im$={a}
+
+$Mary^\Im$={b}
+
+$Jack^\Im$={c}
+
+[6] Explain the difference - using natural language - between the first-order prefixes:
+  ```(a) ∃x∀y and ∀x∃y
+  (b) ∃x∀y∃z and ∀x∃y∀z 
+  (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+```
+Answer:
+
+(a) ∃x∀y means “there exists x such that for all y” while ∀x∃y means "for all x, there exists y such that"
+
+(b) ∃x∀y∃z means “there exists x such that for all y, there exists z such that” while ∀x∃y∀z means "for all x, there exists y such that for all z"
+
+(c) ∀x∃y∀z∃w means "for all x, there exists y such that for all z, there exists w such that" while ∃x∀y∃z∀w means "there exists x such that for all y, there exists z such that for all w"
+
+[7] Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+```
+
+Answer:
+
+The following graph models the second sentence but not the first sentence:
+
+Verticles: a, b, c
+
+Edges: ab, ac, aa (loop)
+
+```mermaid
+graph LR
+A((a)) ---B((b))
+A((a)) ---C((c))
+A((a)) ---A((a))
+```
+	
+[8] Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+```
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+(b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+(c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+(d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+```
+
+Answer:
+
+(a)
+
+![picture 1](https://github.com/peihongx/PHI-696/blob/main/Project-1/tree_proof_1.png)
```

Comment:
What justifies the move from a negated universal in line 1 to a negated universal in line 2?

Reply:
> What justifies the move from a negated universal in line 1 to a negated universal in line 2?

If this move cannot be justified, it must be the case that the online tree proof generator goes wrong because this move is just what the generator does. Anyway, I read G. Priest's _An Introduction to Non-Classical Logic_, and use his tree-constructing rule for quantifers to rewirte these 4 tree proof manully. This must be what you want.